.github/workflows/analytics-metadata-updater.yml

```diff
@@ -0,0 +1,56 @@
+name: CDK Analytics Metadata Updater
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 16 * * 2" # Every week at 10 AM PDT or 9 AM PST
+
+jobs:
+  update-analytics-metadata:
+    if: github.repository == 'aws/aws-cdk'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
```

Comment:
do you need 	`pull-requests: write` to be able to create PR, and add labels to it

Reply:
I think you're right. I haven't figured out a way to test this, but looking at other GHA, it is needed.